static/js/tools/choropleth/choropleth.tsx

```diff
@@ -356,4 +373,38 @@ function determineColorPalette(dict, pc: boolean, popMap: []): number[] {
   return [lowerValue, medianValue, upperValue];
 }
 
+/**
+ * Returns a nicely formatted hover value for the value of statistical variables
+ * for geographies.
+ * @param geoValue statistical variable value to display.
+ * @param isPerCapita whether the value is a per capita indicator.
+ */
+function formatGeoValue(geoValue, isPerCapita) {
+  // Per capita values may be difficult to read so add a per count field.
+  // E.g. 0.0052 or 5.2 per 1000.
+  if (isPerCapita || geoValue === "No Value") {
```

Comment:
Curious why you are testing for "no value" here? The calculations require a numeric value.

Is there a way to simplify this if/else - early returns are more readable imo.

Reply:
Updated to early return.